packages/application/themes/lightTheme.tsx

```diff
@@ -39,13 +39,13 @@ const lightTheme = extendTheme({
         textAlign: 'center',
```

Comment:
Conflicts in this file can be resolved by pasting the entire contents of ligthTheme in this commit and replacing what we have in main. Otherwise, ligthTheme would still have issues.